edge_computing/image-base-install/ibi-preparing-for-image-based-install.adoc

```diff
@@ -0,0 +1,24 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ibi-preparing-for-image-based-install_{context}"]
+= Preparing for an image-based installation for {sno} clusters
+include::_attributes/common-attributes.adoc[]
+:context: ibi-understanding-image-based-install
+
+toc::[]
+To prepare for an image-based installation for {sno} clusters, you must complete the following tasks:
+
+* Install the Image Based Install Operator on the hub cluster.
+* Create a seed image by using the {lcao}.
+* Verify that all software components meet the required versions. 
+
+include::modules/ibi-install-lcao-console.adoc[leveloffset=+1]
+
+include::modules/ibi-install-lcao-cli.adoc[leveloffset=+1]
+
+include::modules/ibi-install-ibi-operator.adoc[leveloffset=+1]
```

Comment:
This is a bit misplaced.
Currently, it's in the middle of the instructions for generating the seed, although it's not relevant to the seed preparation stage.
This section needs to be a prerequisite for https://75088--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image-base-install/ibi-edge-image-based-install

Reply:
Moved to the deploying section.